neqo-http3/src/connection.rs

```diff
@@ -995,6 +998,30 @@ impl Http3Connection {
         Ok(())
     }
 
+    /// Set the stream SendOrder.
+    /// # Errors
+    /// Returns InvalidStreamId if the stream id doesn't exist
+    pub fn stream_set_sendorder(
+        &mut self,
+        conn: &mut Connection,
+        stream_id: StreamId,
+        sendorder: Option<SendOrder>) -> Res<()> {
+        Ok(conn.stream_sendorder(stream_id, sendorder)?)
+    }
+
+    /// Set the stream Fairness.   Fair streams will share bandwidth with other
+    /// streams of the same sendOrder group (or the unordered group).  Unfair streams
+    /// will give bandwidth preferentially to the lowest streamId with data to send.
+    /// # Errors
+    /// Returns InvalidStreamId if the stream id doesn't exist
+    pub fn stream_set_fairness(
+        &mut self,
+        conn: &mut Connection,
+        stream_id: StreamId,
+        fairness: bool) -> Res<()> {
+        Ok(conn.stream_fairness(stream_id, fairness)?)
```

Comment:
As conn.stream_fairness() returns Res, you can do this:

```suggestion
        conn.stream_fairness(stream_id, fairness)
```

There are a lot of other cases where you have the same redundancy.

Reply:
Aha, and here's why I used Ok(...?):
```
error[E0308]: mismatched types
    --> neqo-http3/src/connection.rs:1022:9
     |
1021 |         fairness: bool) -> Res<()> {
     |                            ------- expected `Result<(), Error>` because of return type
1022 |         conn.stream_fairness(stream_id, fairness)
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `Error`, found `neqo_transport::Error`
     |
     = note: `neqo_transport::Error` and `Error` have similar names, but are actually distinct types
note: `neqo_transport::Error` is defined in crate `neqo_transport`
    --> /home/jesup/src/mozilla/neqo/neqo-transport/src/lib.rs:64:1
     |
64   | pub enum Error {
     | ^^^^^^^^^^^^^^
note: `Error` is defined in the current crate
    --> neqo-http3/src/lib.rs:191:1
     |
191  | pub enum Error {
     | ^^^^^^^^^^^^^^
help: use `?` to coerce and return an appropriate `Err`, and wrap the resulting value in `Ok` so the expression remains of type `Result`
     |
1022 |         Ok(conn.stream_fairness(stream_id, fairness)?)
     |         +++                                         ++
```

Reply:
conn.stream_fairness(stream_id, fairness).map_err(|_| Error::InvalidStreamId) works, though likely there's a better solution